ihatemoney/tests/tests.py

```diff
@@ -408,6 +415,8 @@ def test_membership(self):
                 "payer": fred_id,
                 "payed_for": [fred_id],
                 "amount": "25",
+                "converted_amount": "25",
+                "original_currency": "USD",
```

Comment:
I don't understand why, when creating a new bill, we need to define both amount and converted_amount?

Reply:
I fix it so it defaults to USD